src/main/kotlin/io/snyk/plugin/ui/toolwindow/IssueDescriptionPanel.kt

```diff
@@ -0,0 +1,6 @@
+package io.snyk.plugin.ui.toolwindow
+
+/**
+ * marker interface
+ */
+interface IssueDescriptionPanel
```

Comment:
Wouldn't an annotation be better?

Reply:
???
Could you explain it a bit more, please?
PS Using "marker" class is quite usual way to mark classes in Idea code.

Reply:
tried to play a bit with annotations... 
looks like it create less readable and more complicated code compared to that simple-stupid marker interface
https://stackoverflow.com/questions/4365095/detecting-the-present-annotations-within-the-given-object-passed-into-a-construc